ddl/bg_worker.go

```diff
@@ -34,11 +33,8 @@ func (d *ddl) handleBgJobQueue() error {
 	err := kv.RunInNewTxn(d.store, false, func(txn kv.Transaction) error {
 		t := meta.NewMeta(txn)
 		owner, err := d.checkOwner(t, bgJobFlag)
-		if terror.ErrorEqual(err, errNotOwner) {
-			return nil
-		}
 		if err != nil {
```

Comment:
Do you still need to check err is not nil? Or move it to logic of `filtrateError?`